src/app/service/poll/CreatePoll.ts

```diff
@@ -75,14 +112,20 @@ function pollEmbed(poll, polls, EmojiList, id): MessageEmbed {
 			text: id,
 		});
 
-	polls.forEach((option: any, index: number) =>{
-		msgEmbed.addField(option, `${EmojiList[index]} : ${option}\n`, true);
+	poll_options.forEach((option: any, index: number) =>{
+		msgEmbed.addField(option.poll_option_name, `${EmojiList[index]}\n`, true);
 
-		msgEmbed.addField('\u200B', '0', true);
+		msgEmbed.addField('\u200B', '\u200B', true);
 
 		msgEmbed.addField('\u200B', '\u200B', false);
 	});
 
+	const ascii = await createAscii(0);
```

Comment:
ascii still here

Reply:
Yes but it is switched off. Just returning string. It's actually live in the garage. Just waiting with the merge until we have FE up because there might be some changes still needed.